File: BirthDayCandles.py
#!/bin/python

import sys

def birthdayCakeCandles(n, ar):
    ar.sort();
    result = 0;
    for i in range(len(ar)):
        if(ar[i] == ar[len(ar)-1]):
            result += 1
    return result

n = int(raw_input().strip())
ar = map(int, raw_input().strip().split(' '))
result = birthdayCakeCandles(n, ar)
print(result)

File: BirthDayCandles.java
import java.util.Arrays;
import java.util.Scanner;

public class BirthDayCandles {

    static int birthdayCakeCandles(int n, int[] ar) {
        int result = 0;
        
        Arrays.sort(ar);
        
        for(int i : ar){
            if(i == ar[ar.length-1])
                result += 1;
        }
        return result;
    }

    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int n = in.nextInt();
        int[] ar = new int[n];
        for (int ar_i = 0; ar_i < n; ar_i++) {
            ar[ar_i] = in.nextInt();
        }
        int result = birthdayCakeCandles(n, ar);
        System.out.println(result);
    }
}